tests/DoctrineTest/InstantiatorTestAsset/SerializableFinalInternalChildAssset.php

```diff
@@ -0,0 +1,8 @@
+<?php
+declare(strict_types=1);
+
+namespace DoctrineTest\InstantiatorTestAsset;
+
+final class SerializableFinalInternalChildAssset extends \ArrayIterator
```

Comment:
Do you really mean to have that may 's's is `Asset`?

Reply:
oh, dat typo ><

Thanks